examples/examples.ipynb

```diff
@@ -165,11 +166,57 @@
     "\n",
     "solve"
    ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/latex": [
+       "$$ \\displaystyle \\mathrm{numpy}(a) = \\begin{bmatrix} {1} & {2} & {3} & {4}  \\\\ {5} & {6} & {7} & {8}  \\\\ {9} & {10} & {11} & {12}  \\end{bmatrix} $$"
```

Comment:
The function name "numpy" is weird as it may overrides the package name (in some cases). Something else (e.g., `myarray`) is preferred.

Reply:
@odashi I renamed function into myNdarray.